examples/tensorflow-comparison/README.md

```diff
@@ -0,0 +1,13 @@
+# Comparing training of singlenode and Multi-GPU Tensorflow.
+These are the examples which were given in [this](https://saturncloud.io/blog/tensorflow_intro/) blogpost.
+
+## Current Scripts
```

Comment:
```suggestion
**Included Notebooks:**
```

Reply:
Can we rename the notebooks:

`train-model-single-gpu.ipynb`
`train-model-multi-gpu.ipynb`